src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -126,3 +126,8 @@
 (defn parse-profile [unparsed-profile]
   (let [unparsed-values (get unparsed-profile "keyValuePairs")]
     (into {} (map (fn [m] [(keyword (m "key")) (m "value")]) unparsed-values))))
+
+(defn get-id-from-nav-segment [segment]
+  (when-not (clojure.string/blank? segment)
+    (let [[ns n] (clojure.string/split segment #":")]
```

Comment:
many things break if I create a workspace with a colon in its name ... but this is another one of them. (probably best to prevent those characters in workspace names instead of fixing here)